plugins/outputs/cloudwatch/cloudwatch.go

```diff
@@ -530,11 +530,11 @@ func (c *CloudWatch) BuildMetricDatum(point telegraf.Metric) []*cloudwatch.Metri
 }
 
 // Make a list of Dimensions by using a Point's tags. CloudWatch supports up to
-// 10 dimensions per metric so we only keep up to the first 10 alphabetically.
+// 30 dimensions per metric so we only keep up to the first 30 alphabetically.
 // This always includes the "host" tag if it exists.
 func BuildDimensions(mTags map[string]string) []*cloudwatch.Dimension {
 
-	const MaxDimensions = 10
+	const MaxDimensions = 30
```

Comment:
If we're already making changes, instead of just modifying this value can you pull this out to a constant here? That way, we can just refer to it in the test rather than redefine it.